engine/src/test/java/com/redhat/cloud/notifications/EmailTemplatesInDbHelper.java

```diff
@@ -154,7 +154,6 @@ protected String generateEmail(TemplateInstance templateInstance, Map<String, Ob
         String result = templateInstance
             .data("action", Map.of("context", context, "bundle", getBundle(), "timestamp", LocalDateTime.now()))
             .data("environment", environment)
-            .data("user", Map.of("firstName", "John", "lastName", "Doe"))
```

Comment:
We are not using the `user` in v2, but we still have code sending this variable to the templates. We should remove the user in a separate PR.

Reply:
Coindentally, there is a conflict on this file.

Reply:
Thats right, thanks